tests/jenkins/install.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/bash
+
+. /gpfs/bbp.cscs.ch/apps/hpc/jenkins/config/modules.sh
+
+CORENRN_TYPE=$1
+
+if [ ${CORENRN_TYPE} = "GPU" ]
```

Comment:
Use quotes to avoid the comparison to break when there are spaces in the var
`if [ "$CORENRN_TYPE" = "GPU" ]`